mypy.ini

```diff
@@ -89,6 +90,9 @@ ignore_errors = True
 [mypy-aiokafka.*]
 ignore_missing_imports = True
 
+[mypy-confluent_kafka.*]
+ignore_missing_imports = True
```

Comment:
issue: I get that it's a big ask, but I'd really prefer if we can stop introducing new ignores for dependencies. How big of a feat would it be to add stubs only for the parts we use, like we've done with isodate previously? 

https://github.com/Aiven-Open/karapace/tree/main/stubs/

Reply:
It wasn't a herculean effort to do it, I've added it in a separate commit. However, looking at the size of this PR I think it makes sense to split it off into its own follow-up PR.

Reply:
Split it off into https://github.com/Aiven-Open/karapace/pull/760

Reply:
Nice! 👌